src/encoded/static/components/navigation/components/SearchBar.js

```diff
@@ -277,9 +295,9 @@ const SelectItemTypeDropdownBtn = React.memo(function SelectItemTypeDropdownBtn(
         <div className="search-item-type-wrapper">
             <DropdownButton id="search-item-type-selector" size="sm" variant="outline-secondary" disabled={disabled}
                 onSelect={(eventKey, evt) => { onChangeSearchItemType(eventKey); }}
-                title={getDropdownTitleText()}>
+                title={getDropdownTitleText()} onToggle={(isOpen, event) => { onToggleSearchItemType(isOpen, event); }}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX props should not use arrow functions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4833117)